HW15/hw15_02Chess.py

```diff
@@ -0,0 +1,107 @@
+class ChessFigure:
+
+    def __init__(self, color, hor, vert):
+        self.color = color
+        self.hor = hor
+        self.vert = vert
+        self.position_h = 1
+        self.position_v = 1
+
+    @staticmethod
+    def check_position(hor: int, vert: int) -> bool:
+        return True if 1 <= hor <= 8 and 1 <= vert <= 8 else print('outside the field')
+
+    def check_step(self, hor: int, vert: int) -> bool:
+        raise NotImplementedError
+
+    def change_color(self) -> str:
+        return 'black' if self.color == 'white' else 'white'
+
+    def get_move(self, hor: int, vert: int):
+        return self.position_h - hor, self.position_v - vert
```

Comment:
Якщо це зміщення, то навпаки:
```suggestion
        return hor - self.position_h, vert - self.position_v
```